torch/tensor.py

```diff
@@ -371,6 +371,24 @@ def __invert__(self):
     def __hash__(self):
         return id(self)
 
+    def __int__(self):
+        if self.numel() == 1:
+            return int(self.storage()[0])
```

Comment:
This is incorrect if tensor has non-zero data offset. I think the safest way would be to do `self[(0,) * self.ndim()]`

Reply:
Flew over my head:

```
>> x = torch.Tensor([1,2])
>> y = x[1:]
>> int(y)
1
```